app/views/capts/_feed.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="column is-half">
+<% if user_signed_in? %>
+<article class="media box">
+<figure class="media-left">
+<p class="image is-64x64">
+<%= gravatar_image_tag(current_user.email, size: 64, alt: current_user.name) %>
+</p>
+</figure>
+<div class="media-content">
+
+<%= render 'capts/form' %>
+
+</div>
+</article>
+<% end %>
+
+<% @capts.each do | capt | %>
+
+    <div class="box">
+    <article class="media">
+    <div class="media-left">
+    <figure class="image is-64x64">
+    <%= gravatar_image_tag(capt.user.email, size: 64, alt: capt.user.name) %>
+    </figure>
+    </div>
+    <div class="media-content">
+    <div class="content">
+    <strong><%= capt.user.name %></strong> <br />
+    <small><%= capt.user.username %></small><br />
+    <p><%= capt.capt %></p>
+    </div>
+    <% if user_signed_in? && current_user.id == capt.user_id %>
+    <nav class="level">
+    <div class="level-left is-mobile">
+    <%= link_to capt, class: "level-item" do %>
+        <span class="icon"><i class="fa fa-link" aria-hidden="true"></i></span>
+        <% end %>
+        <%= link_to edit_capt_path(capt), class: "level-item" do %>
+            <span class="icon"><i class="fa fa-pencil" aria-hidden="true"></i></span>
+            <% end %>
+                <%= link_to capt, method: :delete, data: { confirm: "Are you sure you want to delete this conversation?" } do %>
+                    <span class="icon"><i class="fa fa-trash-o" aria-hidden="true"></i></span>
+                    <% end %>
+  				</div>
+  			</nav>
+        <% end %>
```

Comment:
- [ ] put the if statement in a helper method that you call here instead and the contents of the if statement in a partial that you render within the helper method to remove the if/end logic from the view